docs-rtd/howto/manage-the-terraform-provider-for-juju.md

```diff
@@ -0,0 +1,128 @@
+(manage-the-terraform-provider-for-juju)=
+# Manage the Terraform Provider for Juju
+
+## Install the Terraform Provider for Juju
+
+To install the Terraform Provider for Juju on Linux, macOS, or Windows, you need to install the `terraform` CLI.
+
+> See more: [Hashicorp | Install Terraform](https://developer.hashicorp.com/terraform/install)
+
+For example, on a Linux that supports snaps:
+
+```text
+sudo snap install terraform
+```
+
+(use-the-terraform-provider-for-juju)=
+## Use the Terraform Provider for Juju
+
+To use the Terraform Provider for Juju, create a Terraform plan specifying the `juju` provider, an existing Juju or JIMM controller, and resources or data sources for whatever Juju entities you want to deploy, then apply your plan in the usual Terraform way.
+
+### 1. Build your Terraform plan
+
+#### a. Configure Terraform to use the `juju` provider
+
+In your Terraform plan, add:
+
+```terraform
+terraform {
+  required_providers {
+    juju = {
+      version = "~> 0.13.0"
```

Comment:
can we please suggest a newer version of the provider? :)

Reply:
Would 0.19.0 be good?

Reply:
yes, absolutely..